website/src/components/SearchPage/DownloadDialog/DownloadUrlGenerator.ts

```diff
@@ -35,28 +35,38 @@ export class DownloadUrlGenerator {
     public generateDownloadUrl(downloadParameters: SequenceFilter, option: DownloadOption) {
         const baseUrl = `${this.lapisUrl}${getEndpoint(option.dataType)}`;
         const params = new URLSearchParams();
+        const excludedParams = new Set<string>();
 
         params.set('downloadAsFile', 'true');
         params.set('downloadFileBasename', this.generateFilename(option.dataType));
+
+        excludedParams.add(VERSION_STATUS_FIELD);
```

Comment:
I'd probably just initialize the set immediately with these 2 params? Bit cleaner :)